integration/tests/cook/test_basic.py

```diff
@@ -237,7 +254,8 @@ def test_change_retries(self):
             '%s/rawscheduler?job=%s' % (self.cook_url, job_uuid)).json()[0]
         self.assertEqual('waiting', job['status'])
         job = self.wait_for_job(job_uuid, 'completed')
-        self.assertEqual('success', job['state'])
+        self.assertEqual('success', job['state'],
```

Comment:
I'm okay extracting these methods, but in general, we have an aversion to extracting the asserts from the unit tests.